File: 04.SentenciasDeControl/Ejercicio1.py
""" Usando un if, crear una condición que compare si la variable numeroIf es positivo, negativo, o 0.
Pista: Los números inferiores a 0 son negativos y los superiores, positivos. """

""" numeroIf = int(input('Ingrese un número entero: '))
if numeroIf == 0:
    print('El número ingresado es cero.')
elif numeroIf < 0:
    print('El número ingresado es negativo.')
else:
    print('El número ingresado es positivo.') """

""" Crea un bucle While, este bucle tendrá que tener como condición que la variable numeroWhile sea inferior a 3, el bloque de código que tendrá el bucle deberá:
    * Incrementar el valor de la variable en uno cada vez que se ejecute.
    * Mostrarlo por pantalla cada vez que se ejecute. """

""" numeroWhile = int(input('Ingrese un número entero menor que 3: '))
if numeroWhile < 3:
    while numeroWhile < 3:
        print(numeroWhile)
        numeroWhile = numeroWhile + 1
else:
    print('El número ingresado no es menor que 3.') """

""" Para el bucle Do While, deberás crear la misma estructura que en el While, pero solo se debe ejecutar una vez. """

""" Para el bucle For, crea una variable numeroFor, esta variable tendrá como valor 0 y su condición será que la variable sea igual o menor que 3, se irá incrementando en 1 su valor cada vez que se ejecute y deberá mostrarse por pantalla. """

numeroFor = 0
while numeroFor <= 3:
    print(numeroFor)
    numeroFor = numeroFor + 1 

File: 04.SentenciasDeControl/Ejercicio1.js
/* Para el bucle Do While, deberás crear la misma estructura que en el While, pero solo se debe ejecutar una vez. */

/* let numeroWhile = parseInt(prompt('Ingrese un número entero menor que 3: '));
if (numeroWhile < 3){
    do {console.log(numeroWhile)
    numeroWhile = numeroWhile + 1}
    while (numeroWhile < 3)
}else{
    console.log('El número ingresado no es menor que 3.')
}; */

/* Por último, para el Switch, deberás crear la variable estacion, y distintos case para las cuatro estaciones del año. Dependiendo del valor de la variable estacion se deberá mandar un mensaje por consola informando de la estación en la que está. También habrá que poner un default para cuando el valor de la variable no sea una estación.  */

let estacion = prompt('¿En qué estación del año estamos?: ')
switch (estacion){
    case 'Primavera':
        console.log('Esperemos que crezcan muchas flores.')
        break;
    case 'Verano':
        console.log('Esperemos que no haga tanto calor.')
        break;
    case 'Otoño':
        console.log('Tenemos que ir preparándonos para el invierno.')
        break;
    case 'Invierno':
        console.log('Esperemos que no haga tanto frío.')
        break;
    default:
        console.log('Usted no ha ingresado una estación del año.')
};
